test/ui/test_ui_maintenance.py

```
@@ -35,33 +31,43 @@ def test_export_csv_button_works(driver):
         os.remove(f)
 
-    # Find the export button
-    export_btns = driver.find_elements(By.ID, "btnExportCSV")
+    # Ensure the Backup/Restore tab is active so the button is in a clickable state
+    try:
+        tab = WebDriverWait(driver, 5).until(
+            EC.element_to_be_clickable((By.ID, "tab_BackupRestore_id"))
+        )
+        tab.click()
+    except Exception:
+        pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)